seventh batch of nls messages for jakarta data

njr-11 opened the pull request:
Various NLS messages for Jakarta Data, including some related to configuration and some related to incompatible results.

- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

njr-11 committed:
Issue #28595 another batch of Jakarta Data messages

njr-11 committed:
Issue #28595 config related messages

njr-11 commented:
#build

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1062.resource.not.found=CWWKD1062E: The {0} method of the {1} repository \
+ interface is unable to locate the {2} resource with the {3} filter.
+CWWKD1062.resource.not.found.explanation=The resource is not found.
+CWWKD1062.resource.not.found.useraction=Ensure that the resource and all other \
```
```suggestion
CWWKD1062.resource.not.found.useraction=Ensure that the specified resource and all other \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ interface is unable to locate the {2} resource with the {3} filter.
+CWWKD1062.resource.not.found.explanation=The resource is not found.
+CWWKD1062.resource.not.found.useraction=Ensure that the resource and all other \
+ resources upon which it depends exist and are correctly configured.
```
```suggestion
 resources it depends on exist and are correctly configured.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ keyword of the Jakarta Data Query by Method Name pattern.
+CWWKD1056.unsupported.keyword.explanation=The keyword is not supported in the \
+ name of a repository method.
+CWWKD1056.unsupported.keyword.useraction=Rename the repository method to not use \
```
```suggestion
CWWKD1056.unsupported.keyword.useraction=Rename the repository method to remove \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ one of the supported types.
+
+CWWKD1056.unsupported.keyword=CWWKD1056E: The {0} method of the {1} repository \
+ interface has a name that includes a keyword, {2}, that is not a supported \
```
```suggestion
 interface has a name that includes a keyword {2}, that is not a supported \
```

njr-11 replied:
I'm fine with removing the first comma, but if we are keeping the remaining comma, then I think we should switch from "that" to "which", otherwise the remaining comma seems out of place.

ramkumar-k-9286 replied:
Agree

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1056.unsupported.keyword=CWWKD1056E: The {0} method of the {1} repository \
+ interface has a name that includes a keyword, {2}, that is not a supported \
+ keyword of the Jakarta Data Query by Method Name pattern.
+CWWKD1056.unsupported.keyword.explanation=The keyword is not supported in the \
```
```suggestion
CWWKD1056.unsupported.keyword.explanation=The method name includes a keyword that is not recognized as part of the \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ interface has a name that includes a keyword, {2}, that is not a supported \
+ keyword of the Jakarta Data Query by Method Name pattern.
+CWWKD1056.unsupported.keyword.explanation=The keyword is not supported in the \
+ name of a repository method.
```
```suggestion
 supported Jakarta Data Query by Method Name pattern.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1057.no.args.constr.inacc.explanation=A public constructor without parameters \
+ is required in order to convert the result to the specified class.
+CWWKD1057.no.args.constr.inacc.useraction=Update the repository method to have a \
+ return type that is supported by Jakarta Data or is a subclass of Iterable with \
```
```suggestion
 return type that Jakarta Data supports or to be a subclass of Iterable \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ is required in order to convert the result to the specified class.
+CWWKD1057.no.args.constr.inacc.useraction=Update the repository method to have a \
+ return type that is supported by Jakarta Data or is a subclass of Iterable with \
+ a public constructor that does not have parameters.
```
```suggestion
 with a public constructor that does not have parameters.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ of the {3} repository interface is not valid for a repository method because the \
+ {4} class lacks a public constructor without parameters.
+CWWKD1057.no.args.constr.inacc.explanation=A public constructor without parameters \
+ is required in order to convert the result to the specified class.
```
```suggestion
 is needed to convert the result to the specified class.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ the unsupported keyword.
+
+CWWKD1057.no.args.constr.inacc=CWWKD1057E: The {0} return type of the {2} method \
+ of the {3} repository interface is not valid for a repository method because the \
```
```suggestion
 in the {3} repository interface is not valid for a repository method because the \
```

njr-11 replied:
Using the word "in" here rather than "of" seems fine. However, it would be inconsistent with the 39 occurrences of the same phrase, "{0} method of the {1} repository", that appear within this same NLS messages file prior to this point, all of which passed reviews from ID team members.  It seems wrong to change only this one.  We should either leave this one consistent with the others is or change them all.

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1058.no.args.constr.err.explanation=The public constructor without parameters \
+ failed with an error.
+CWWKD1058.no.args.constr.err.useraction=Correct the cause of the error, or update \
+ the repository method to have a different return type.
```
```suggestion
 the repository method to use a different return type.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ of the {3} repository interface is not valid for a repository method because the \
+ constructor of the {4} class raised an error: {5}.
+CWWKD1058.no.args.constr.err.explanation=The public constructor without parameters \
+ failed with an error.
```
```suggestion
 failed due to an error.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ a public constructor that does not have parameters.
+
+CWWKD1058.no.args.constr.err=CWWKD1058E: The {0} return type of the {2} method \
+ of the {3} repository interface is not valid for a repository method because the \
```
```suggestion
 in the {3} repository interface is not valid for a repository method because the \
```

njr-11 replied:
See previous comment on the precedent set in prior messages for "of" vs "in".

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ constructor of the {4} class raised an error: {5}.
+CWWKD1058.no.args.constr.err.explanation=The public constructor without parameters \
+ failed with an error.
+CWWKD1058.no.args.constr.err.useraction=Correct the cause of the error, or update \
```
```suggestion
CWWKD1058.no.args.constr.err.useraction=Fix the error in the constructor, or update \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+
+CWWKD1059.prop.cast.err=CWWKD1059E: The {0} method of the {1} repository \
+ interface cannot obtain the {2} property of the {3} entity because the \
+ {4} field or method of the {5} class cannot be used on a {6} value.
```
```suggestion
 {4} field or method of the {5} class is incompatible with a {6} value.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1058.no.args.constr.err.useraction=Correct the cause of the error, or update \
+ the repository method to have a different return type.
+
+CWWKD1059.prop.cast.err=CWWKD1059E: The {0} method of the {1} repository \
```
```suggestion
CWWKD1059.prop.cast.err=CWWKD1059E: The {0} method in the {1} repository \
```

njr-11 replied:
See prior comment about "of" vs "in".

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ the repository method to have a different return type.
+
+CWWKD1059.prop.cast.err=CWWKD1059E: The {0} method of the {1} repository \
+ interface cannot obtain the {2} property of the {3} entity because the \
```
```suggestion
 interface cannot access the {2} property of the {3} entity because the \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1059.prop.cast.err=CWWKD1059E: The {0} method of the {1} repository \
+ interface cannot obtain the {2} property of the {3} entity because the \
+ {4} field or method of the {5} class cannot be used on a {6} value.
+CWWKD1059.prop.cast.err.explanation=An entity property or a property from its \
```
```suggestion
CWWKD1059.prop.cast.err.explanation=An entity property or a related property from its \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ interface cannot obtain the {2} property of the {3} entity because the \
+ {4} field or method of the {5} class cannot be used on a {6} value.
+CWWKD1059.prop.cast.err.explanation=An entity property or a property from its \
+ hierarchy of relations is not compatible the expected type.
```
```suggestion
 hierarchy is not compatible with the expected type.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ {4} field or method of the {5} class cannot be used on a {6} value.
+CWWKD1059.prop.cast.err.explanation=An entity property or a property from its \
+ hierarchy of relations is not compatible the expected type.
+CWWKD1059.prop.cast.err.useraction=Review the entity model and ensure that the \
```
```suggestion
CWWKD1059.prop.cast.err.useraction=Review the entity model and verify that the \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1059.prop.cast.err.explanation=An entity property or a property from its \
+ hierarchy of relations is not compatible the expected type.
+CWWKD1059.prop.cast.err.useraction=Review the entity model and ensure that the \
+ expected types are used for each property.
```
```suggestion
 correct types are used for each property.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1059.prop.cast.err.useraction=Review the entity model and ensure that the \
+ expected types are used for each property.
+
+CWWKD1060.name.out.of.scope=CWWKD1060E: The {0} repository interface from the \
```
```suggestion
CWWKD1060.name.out.of.scope=CWWKD1060E: The {0} repository interface in the \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ expected types are used for each property.
+
+CWWKD1060.name.out.of.scope=CWWKD1060E: The {0} repository interface from the \
+ {1} application configures a {2} value for the {3} attribute, but {3} names \
```
```suggestion
 {1} application configures a {2} value for the {3} attribute, but {3} names \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ {1} application configures a {2} value for the {3} attribute, but {3} names \
+ are not accessible to the artifact that provides the repository interface. \
+ Use a {4} name instead.
+CWWKD1060.name.out.of.scope.explanation=The specified namespace is not visible \
```
```suggestion
CWWKD1060.name.out.of.scope.explanation=The specified namespace is not accessible \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ are not accessible to the artifact that provides the repository interface. \
+ Use a {4} name instead.
+CWWKD1060.name.out.of.scope.explanation=The specified namespace is not visible \
+ to the artifact in which the repository interface is defined.
```
```suggestion
 from the artifact where the repository interface is defined.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1060.name.out.of.scope.explanation=The specified namespace is not visible \
+ to the artifact in which the repository interface is defined.
+CWWKD1060.name.out.of.scope.useraction=Update the Repository annotation to \
+ specify a data store name that is visible to the module or library that \
```
```suggestion
 specify a data store name that is visible to the module or library \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ to the artifact in which the repository interface is defined.
+CWWKD1060.name.out.of.scope.useraction=Update the Repository annotation to \
+ specify a data store name that is visible to the module or library that \
+ provides the repository interface.
```
```suggestion
 providing the repository interface.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ specify a data store name that is visible to the module or library that \
+ provides the repository interface.
+
+CWWKD1061.comp.name.in.ejb=CWWKD1061E: The {0} repository interface that is \
```
```suggestion
CWWKD1061.comp.name.in.ejb=CWWKD1061E: The {0} repository interface which is \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+
+CWWKD1061.comp.name.in.ejb=CWWKD1061E: The {0} repository interface that is \
+ defined in the {1} enterprise bean module of the {2} application configures \
+ a {2} value for the {3} attribute, but {4} names are not accessible to \
```
```suggestion
 a {2} value for the {3} attribute. However,  {4} names are not accessible to \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1061.comp.name.in.ejb=CWWKD1061E: The {0} repository interface that is \
+ defined in the {1} enterprise bean module of the {2} application configures \
+ a {2} value for the {3} attribute, but {4} names are not accessible to \
+ enterprise bean modules. Switch to a name in one of the following \
```
```suggestion
 enterprise bean modules. Use a name from one of the following \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ a {2} value for the {3} attribute, but {4} names are not accessible to \
+ enterprise bean modules. Switch to a name in one of the following \
+ namespaces: {5}.
+CWWKD1061.comp.name.in.ejb.explanation=The specified namespace is not visible \
```
```suggestion
CWWKD1061.comp.name.in.ejb.explanation=The specified namespace is not accessible \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ enterprise bean modules. Switch to a name in one of the following \
+ namespaces: {5}.
+CWWKD1061.comp.name.in.ejb.explanation=The specified namespace is not visible \
+ to the enterprise bean module in which the repository interface is defined.
```
```suggestion
 to the enterprise bean module where the repository interface is defined.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1062.resource.not.found.useraction=Ensure that the resource and all other \
+ resources upon which it depends exist and are correctly configured.
+
+CWWKD1063.unsupported.resource=CWWKD1063E: The {0} method of the {1} repository \
```
```suggestion
CWWKD1063.unsupported.resource=CWWKD1063E: The {0} method in the {1} repository \
```

njr-11 replied:
See prior comment about "of" vs "in".

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+
+CWWKD1063.unsupported.resource=CWWKD1063E: The {0} method of the {1} repository \
+ interface cannot obtain a {2} resource because the Jakarta Persistence provider \
+ does not support unwrapping as a {3} class.
```
```suggestion
 does not support unwrapping it as a {3} class.
```

njr-11 replied:
The proposed change here would result in a meaning that is not correct. I'll need to try writing this differently.

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1063.unsupported.resource=CWWKD1063E: The {0} method of the {1} repository \
+ interface cannot obtain a {2} resource because the Jakarta Persistence provider \
+ does not support unwrapping as a {3} class.
+CWWKD1063.unsupported.resource.explanation=The requested type of resource is not \
```
```suggestion
CWWKD1063.unsupported.resource.explanation=The Jakarta Persistence provider does not \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ interface cannot obtain a {2} resource because the Jakarta Persistence provider \
+ does not support unwrapping as a {3} class.
+CWWKD1063.unsupported.resource.explanation=The requested type of resource is not \
+ available from the Jakarta Persistence provider.
```
```suggestion
 support unwrapping the requested resource type.
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+CWWKD1055.unsupported.entity.prop=CWWKD1055E: The {0} property of the {1} entity \
+ has an unsupported type: {2}. Jakarta Data supports Java enumeration types, \
+ the {3} temporal types, and the following basic types: {4}. The built-in \
+ Jakarta Data provider also allows some Jakarta Persistence types, such as \
```
```suggestion
 Jakarta Data provider also supports some Jakarta Persistence types, such as \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ the {3} temporal types, and the following basic types: {4}. The built-in \
+ Jakarta Data provider also allows some Jakarta Persistence types, such as \
+ embeddables.
+CWWKD1055.unsupported.entity.prop.explanation=The entity property has a type \
```
```suggestion
CWWKD1055.unsupported.entity.prop.explanation=Jakarta Data does not support the type \
```

ramkumar-k-9286 commented on dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops:
```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
+ Jakarta Data provider also allows some Jakarta Persistence types, such as \
+ embeddables.
+CWWKD1055.unsupported.entity.prop.explanation=The entity property has a type \
+ that is not supported in Jakarta Data.
```
```suggestion
 used by the entity property.
```

njr-11 committed:
Apply suggestions from code review

Applying most of the suggestions as recommended. I'll need to address a couple of the remaining ones differently. We will also need a separate resolution to the usage of "of" vs "in" which needs to be consistent across the entire messages file.

Co-authored-by: Ramkumar <88021034+ramkumar-k-9286@users.noreply.github.com>

njr-11 committed:
Fix incorrect numbering of message parameters

njr-11 committed:
Alternative fixes to 2 NLS msesage review issues

njr-11 commented:
@ramkumar-k-9286 I added almost all of your suggestions.  Two that I correctly differently are in commit https://github.com/OpenLiberty/open-liberty/pull/29628/commits/f3d774053cf645ab3c821a963b18f7fc7088d8c6
Beyond that, the only remaining suggestions that I haven't included are the ones to switch "of" to "in", which would be inconsistent with the 39 prior occurrences of the same phrase in the file.

njr-11 commented:
#build
#libby

njr-11 commented:
> https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?uuid=0091ec11-00ef-4fd0-ad18-aeb416910e08 has errors/failures.

The failures are pre-existing issues in unrelated test buckets.
